tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPReplicaTest.java

```diff
@@ -845,23 +852,31 @@ private void consumeMessages(boolean largeMessage,
                                 int LAST_ID,
                                 int port,
                                 boolean assertNull) throws JMSException {
-      ConnectionFactory cf = CFUtil.createConnectionFactory("AMQP", "tcp://localhost:" + port);
+      ConnectionFactory cf = CFUtil.createConnectionFactory("AMQP", "tcp://localhost:" + port + "?jms.prefetchPolicy.all=0");
```

Comment:
Is this needed when there is only one consumer at a time?

Reply:
It would be my preference. especially because of cancelling deliveries in paging.